.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,45 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*      
+    - name: Update version for next release
```

Comment:
we need to sync on version numbering too, we're currently using calendar versioning not semantic

Reply:
i have very little memory writing this, but what if we stick your the calendar versioning, with an auto generated patch number if we release more then once in a month?
2021.2.0
2021.2.1
2021.3.0
?

Reply:
works for me, surely github actions have a version incrementing scheme?